sky/cli.py

```diff
@@ -3062,13 +3063,20 @@ def _output():
                        '(including non-common ones) and pricing.')
                 return
 
+        new_gpu_name = gpu_name
+        gpu_name_and_count = new_gpu_name.count(':') > 0
```

Comment:
Can we directly split the gpu_str using `gpu_name.split(':')` and then check the length of the output to see and extract the count?
```
>>> "V100".split(':')
['V100']
>>> "V100:4".split(':')
['V100', '4']
```

Reply:
That way we don't have to `import re`.